charts/sn-platform/values.yaml

```diff
@@ -2353,4 +2364,9 @@ istio:
     # gateway selector if it's not `istio: ingressgateway`
     selector: {}
     tls:
+      # use existing secret
       certSecretName: "[CERTIFICATE SECRET NAME IN ISTIO ROOT NAMESPACE]"
+      # use cert-manager to generate secret, only take effect if certSecretName is empty,
+      # also need to config certs.istio_internal_issuer/certs.istio_public_issuer
+      cert_name: "[ISTIO CERT NAME]"
+      cert_namespace: "[ISTIO ROOT NAMESPACE]"
```

Comment:
Should we leave the value empty otherwise the default value (`istio-system`) will not work?

Reply:
> Seems camelcase is prefered for helm chart value names: https://helm.sh/docs/chart_best_practices/values/#naming-conventions

Follow eric's suggestion, I refer to the previous cert-related variables, should we fix all these values?
https://github.com/streamnative/charts/blob/91e4dcc9bbc1c50b5c91017e0ae569c209725d3b/charts/sn-platform/values.yaml#L283
https://github.com/streamnative/charts/blob/91e4dcc9bbc1c50b5c91017e0ae569c209725d3b/charts/sn-platform/values.yaml#L291

Reply:
> Should we leave the value empty otherwise the default value (`istio-system`) will not work?

Yes, I will leave it empty.

Reply:
> Follow eric's suggestion, I refer to the previous cert-related variables, should we fix all these values?

No need to rename existing names as it might cause compatibility issues, we just need to make it consistent for new names for now.